documentation/en/default-lotus-miner-config.toml

```diff
@@ -666,9 +666,9 @@
   # env var: LOTUS_SEALING_TERMINATEBATCHWAIT
   #TerminateBatchWait = "5m0s"
 
-  # UseSyntheticPoRep will reduce the amout of data held on disk in the WaitSeed phase to 32GiB
-  # at the cost of having to precompute the synthetic challenges.
-  # TODO: put estimated costs here
+  # UseSyntheticPoRep, when set to true, will reduce the amount of cache data held on disk after the completion of PreCommit 2 to 11GiB.
+  # This is achieved by precomputing "synthetic" challenges from CommR.
+  # Note that this comes at a cost of additional computation to generate these synthetic challenges.
```

Comment:
Do we have some number to put on this as a rough guidance?

Reply:
I do not have any numbers right now, but the additional costs should be minimal. I can see if we can get a SynthPoRep vs non SynthPoRep sealing benchmark done

Reply:
As discussed, removed:
> Note that this comes at a cost of additional computation to generate these synthetic challenges.

Given that costs are negligible, and we currently do not have any good estimations on the diff here between SynthPoReps vs Non-SynthPoRep